model/beacon/beacon-facade.js

```diff
@@ -0,0 +1,15 @@
+const Model = require('../../lib/facade');
+const beaconSchema  = require('./beacon-schema');
+
+class BeaconModel extends Model {
+
+  updateBeacons(beaconsList) {
+    return beaconSchema
+      .remove({})
```

Comment:
This method has been created to update the entire list of beacon from the website proximy
So first I have to remove all beacons in db